lib/cisco_node_utils/vrf_af.rb

```diff
@@ -83,6 +83,40 @@ def route_target_feature_enable
     #                      PROPERTIES                      #
     ########################################################
 
+    def route_policy_export
+      config_get('vrf', 'route_policy_export', @get_args)
+    end
+
+    def route_policy_export=(name)
+      if platform == :nexus
+        fail Cisco::UnsupportedError.new('vrf_af', route_policy_export)
```

Comment:
Shouldn't be needed - the config_set should automatically fail and raise this error.